### DISCUSSION TITLE
How can I do a backup of my Profiles

### DISCUSSION THREAD

**Discussion by papa-pitufo** [staxrip/staxrip#758] [General]
Hi everybody.

First, congratulations for the new version 2.7.0

But I read:
_Changed names cause all encoder profiles to be reset. This means you have to re-set them by hand or backup and restore them after update._

I understand that I don't need to re write all of my Profiles if I do a backup.

Usually I extract the new release in your own folder, and copy my old Settings folder.  On this way I get all my old profiles.
If only want my old templates, is easiest, You only need to copy template folder.

I think, the profiles are inside Settings.dat (only I think), therefore nowadays It's impossible to create a folder like Templates. Isn't it?
Is there an option to update the program and later bring your old profiles?

Best regards and congratulations again

**Comment by Dendraspis** [comment]
Those warnings/information are just short notices so you know where you have to pay attention. Most the times when I write them, the implementation is not finished yet and I also don't want to write a book about the best solution to deal with the issue for every single case. In that case it is a bit misleading, because normally old settings should be backed up, but they don't.

Here the best way is to start with fresh settings, as long as you don't have 20 custom filters, 20 events, ....
Just start the old and new versions in parallel and restore your old settings by hand.

**Reply by papa-pitufo** [reply]
Thanks, that's what I usually do, copy them by hand.

Best regards
